dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/tasks/CreateConnectionTask.java

```diff
@@ -32,6 +32,7 @@ public static Session createConnection(final NewSessionModel newSessionModel, fi
                             new Credentials(newSessionModel.getAccessKey(), newSessionModel.getSecretKey()))
                     .withHttps(newSessionModel.isUseSSL()).withCertificateVerification(false)
                     .withProxy(newSessionModel.getProxyServer())
```

Comment:
A couple thoughts...

1) Can't really tell from the GitHub diff, but does this file have a copyright comment?
2) Should the string "Eon Browser" be in a resource bundle or maybe a macro?  Guessing that that string might show up in other places, like title bar text, but I don't know that.
3) Should there be a unit test to make sure we're picking up the build correctly?